### DISCUSSION TITLE
How would i make it so that when paused and reset or quit is pressed

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#160] [Q&A]
I have a pop up that appears when restart or quit is pressed but its supposed to have input but it doesn't 
[PauseManager2.gd.zip](https://github.com/user-attachments/files/15750623/PauseManager2.gd.zip)

**Comment by r6915ee** [comment]
Are you sure you included the input within the popup?

**Reply by Raptorq1** [reply]
Its on top of the pause menu and the pause menu still has input

**Reply by NeonSRB2** [reply]
Try checking the process mode for the menu scene's root node, it should be set to "When Paused". Otherwise the menu itself is paused and won't do anything, at all, let alone ignore input.

**Reply by Raptorq1** [reply]
I already did that but still the same thing is happening
